R/utils-stats.R

```diff
@@ -1,3 +1,33 @@
+medianCI <- function(x, conf.level=.95) {
+  w <- try(wilcox.test(x, conf.int=T, conf.level=conf.level))
```

Comment:
My stats is admittedly weak. Curious why use wilcox.test to compute CIs for medians when we could use a direct [formula](https://www-users.york.ac.uk/~mb55/intro/cicent.htm)? Is it the lack of normality assumption that makes it a winner for this case?
I also ask because `wilcox.test` is also computing things we don't need, and it could be faster to compute the CIs directly.

Reply:
.. bc it was easy to do and meant i didnt have to think too hard and i was feeling a bit lazy. a little i was also inclined to making the fxn as consistent w the other CI fxns as possible. if it turns out to be a performance problem ill revisit it, but i dont think im going to over optimize at this point.